### ISSUE TITLE
Show "Installed yesterday" instead of "Installed 1 day ago" (Align iOS to Android)

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-app-ios#3648]
## Avoid duplicates
* [X] This enhancement request has not already been raised before
* [X] Enhancement request is specific for iOS only, for general issues / questions that apply to iOS and Android please raise them in [CWA-Wishlist](https://github.com/corona-warn-app/cwa-wishlist)
* [X] If you are proposing a new feature, please do so in [CWA-Wishlist](https://github.com/corona-warn-app/cwa-wishlist)

## Current Implementation

| iOS | Android |
|---|---|
| <img src="https://user-images.githubusercontent.com/67682506/136083450-db39c6c2-abaf-4be0-8429-4361e74e6b58.png" width="300"> | <img src="https://user-images.githubusercontent.com/67682506/136083446-10abc99e-3b54-4489-a25a-25daddcaf8d7.png" width="300"> |

iOS shows "Installed 1 day ago" while Android shows "Installed yesterday".

## Suggested Enhancement
"Installed yesterday" sounds better than "Installed 1 day ago". iOS should therefore be aligned to Android.

## Expected Benefits
Better UX.

**Comment by Ein-Tim** [comment]
> @Ein-Tim Thanks. Did you check also German?

I can't check German under Android, but iOS shows "Installiert seit 1 Tag" in German.

**Comment by dsarkar** [comment]
@Ein-Tim ok, thanks: Android shows "yesterday"
![image](https://user-images.githubusercontent.com/1409741/136160003-6cfaa313-b010-4e61-85bc-f091f1e17a4b.png)

**Comment by mtwalli** [comment]
@naveeddotio  or @30mar  can you transfer this issue to Android Repo ? I have no access here

**Comment by Ein-Tim** [comment]
@mtwalli For the record Android was now aligned to iOS, right? :D

**Comment by mtwalli** [comment]
> @mtwalli For the record Android was now aligned to iOS, right? :D

Yes, according to latest comments on the ticket `"Installed 1 day ago"`  is the right wordings
